style.css

```diff
@@ -349,3 +363,250 @@ input.form-control::placeholder {
   background: #0d151e;
   border-radius: 10px;
 }
+
+/* Media-Queries */
+@media screen and (min-width: 1044px) {
+  body {
+    font-family: "Poppins", sans-serif;
+  }
+
+  .header {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    background-color: #fff;
+  }
+
+  .header .bars {
+    display: none;
+  }
+
+  .logo a {
+    padding-left: 30px;
+    margin-left: 55px;
+  }
+
+  .menu-content ul {
+    display: flex;
+    align-items: center;
+    text-align: center;
+    list-style: none;
+    padding-right: 45px;
+  }
+
+  .menu-content li {
+    padding: 1rem;
+  }
+
+  .menu-content a {
+    text-decoration: none;
+    font-size: 15px;
+    font-weight: 500;
+    color: #344563;
+  }
+
+  .menu-content {
+    display: flex;
+  }
+
+  .intro {
+    background-image: url(./img/header_bg-desktop.svg);
+    background-repeat: no-repeat;
+    background-size: contain;
+    background-color: #fff;
+    background-position: center;
+  }
+
+  .intro h2 {
+    color: #091e42;
+    letter-spacing: 0.37px;
+    padding-left: 120px;
+    margin: 180px 0 15px 0;
+    font-size: 48px;
+    line-height: 60px;
+  }
+
+  .intro p {
+    margin: 0 35px 0 120px;
+  }
+
+  .connect {
+    padding-left: 122px;
+    margin-top: -30px;
+  }
+
+  .socials {
+    display: flex;
+    justify-content: space-between;
+    width: 13%;
+    padding-top: 15px;
+    margin-bottom: 100px;
+    margin-left: 123px;
+  }
```

Comment:
- [ ] Kindly adjust the spacing between each social media icon below as requested by the previous reviewer.
- Currently it's not displaying at around 768px - 1000px view. Please fix that 👍

** Your Design **
![image](https://user-images.githubusercontent.com/50941074/160216449-5657f31c-a7a6-473b-b8fd-eb3d3cba345a.png)

** The Figma Design **
![image](https://user-images.githubusercontent.com/50941074/160216482-007236bf-8a2b-4611-9a25-e60db97a3db4.png)